traitlets/utils/bunch.py

```diff
@@ -23,7 +23,6 @@ def __setattr__(self, key: str, value: Any) -> None:
         self.__setitem__(key, value)
 
     def __dir__(self) -> list[str]:
-        # py2-compat: can't use super because dict doesn't have __dir__
-        names = dir({})
+        names = dir(Bunch)
```

Comment:
This should be the right super invocation, now that we can assume recent Python:

```suggestion
        names = super().__dir__()
```